src/main/kotlin/uk/gov/justice/digital/hmpps/deliusapi/security/ProviderResponseAuthority.kt

```
@@ -1,8 +0,0 @@
-package uk.gov.justice.digital.hmpps.deliusapi.security
-
-import org.springframework.security.access.prepost.PostAuthorize
-import uk.gov.justice.digital.hmpps.deliusapi.config.Authorities
-
-@Retention(AnnotationRetention.RUNTIME)
-@PostAuthorize("hasAuthority('${Authorities.PROVIDER}'.concat(returnObject.provider))")
-annotation class ProviderResponseAuthority
```

This didnt actually work.. The failure was masked by the fact that the only place it was used was in a private function, which is ignored by AOP security.